<?xml version="1.0" encoding="utf-8" ?>
<!-- This URDF was automatically created by SolidWorks to URDF Exporter! Originally created by Stephen Brawner (brawner@gmail.com) 
     Commit Version: 1.6.0-4-g7f85cfe  Build Version: 1.6.7995.38578
     For more information, please see http://wiki.ros.org/sw_urdf_exporter -->
<robot name="Arm_Urdf">
    <material name="black">
        <color rgba="0.15 0.15 0.15 1.0"/>
    </material>
    <link name="base_link">
        <inertial>
            <origin xyz="-0.0108454625152952 5.41174924359317E-05 0.006809784336333" rpy="0 0 0" />
            <mass value="0.77116648882986" />
            <inertia ixx="0.000698139677914449" ixy="-9.98098781596937E-09" ixz="-4.42589020292821E-09" iyy="0.000247085234791679" iyz="-7.75794099442475E-09" izz="0.000718421438540256" />
        </inertial>
        <visual>
            <origin xyz="0 0 0" rpy="0 0 0" />
            <geometry>
                <mesh filename="package://Arm_Urdf/meshes/base_link.STL" />
            </geometry>
            <material name="">
                <color rgba="0.650980392156863 0.619607843137255 0.588235294117647 1" />
            </material>
        </visual>
        <collision>
            <origin xyz="0 0 0" rpy="0 0 0" />
            <geometry>
                <mesh filename="package://Arm_Urdf/meshes/base_link.STL" />
            </geometry>
        </collision>
    </link>
    <link name="world" />
    <joint name="world_joint" type="fixed">
        <origin xyz="0 0 0.0" rpy="0 0 0" />
        <parent link="world" />
        <child link="base_link" />
    </joint>
    <link name="Link_1">
        <inertial>
            <origin xyz="0.00202559488051276 0.0292417340051791 0.0299779266003479" rpy="0 0 0" />
            <mass value="2.19576081627391" />
            <inertia ixx="0.00622314668686051" ixy="2.69943075030139E-08" ixz="-1.96744672740614E-07" iyy="0.00460468862088576" iyz="-6.43893757821027E-05" izz="0.00970799857789177" />
        </inertial>
        <visual>
            <origin xyz="0 0 0" rpy="0 0 0" />
            <geometry>
                <mesh filename="package://Arm_Urdf/meshes/Link_1.STL" />
            </geometry>
            <material name="">
                <color rgba="0.529411764705882 0.549019607843137 0.549019607843137 1" />
            </material>
        </visual>
        <collision>
            <origin xyz="0 0 0" rpy="0 0 0" />
            <geometry>
                <mesh filename="package://Arm_Urdf/meshes/Link_1.STL" />
            </geometry>
        </collision>
    </link>
    <joint name="Joint_1" type="revolute">
        <origin xyz="0 0 0.073" rpy="0 0 0" />
        <parent link="base_link" />
        <child link="Link_1" />
        <axis xyz="0 0 1" />
        <limit lower="-3.17" upper="3.17" effort="50" velocity="5" />
        <dynamics damping="0.5" friction="0.1"/>
    </joint>

    <!-- camera base left -->


    <!-- <joint name="camera_joint_base_left" type="fixed">
        <axis xyz="0 1 0"/>
        <origin rpy="0 0 -1.57" xyz="0.12 -0.15 0.02"/>
        <parent link="Link_1"/>
        <child link="camera_link_base_left"/>
    </joint>


    <link name="camera_link_base_left">
        <inertial>
            <mass value="1e-5"/>
            <origin rpy="0 0 0" xyz="0 -0.02 0.02"/>
            <inertia ixx="1e-6" ixy="0" ixz="0" iyy="1e-6" iyz="0" izz="1e-6"/>
        </inertial>
        <collision>
            <origin rpy="0 0 -1.570" xyz="0 0 0"/>
            <geometry>
                <box size="0.025 0.025 0.025"/>
            </geometry>
        </collision>
        <visual>
            <origin rpy="0 0 -1.570" xyz="0 0 0"/>
            <geometry>
                <box size="0.025 0.025 0.025"/>
            </geometry>
            <material name="black">
                <color rgba="1 1 1 1"/>
            </material>
        </visual>
    </link>

    <joint name="camera_optical_joint_base_left" type="fixed">
        <parent link="camera_link_base_left"/>
        <child link="camera_link_optical_base_left"/>
        <origin xyz="0 0 0" rpy="-1.5707963 0 -1.5707963"/>
    </joint>

    <link name="camera_link_optical_base_left"></link> -->


    <!-- camera base right -->


    <!-- <joint name="camera_base_joint_right" type="fixed">
        <axis xyz="0 1 0"/>
        <origin rpy="0 0 -1.57" xyz="-0.12 -0.15 0.1"/>
        <parent link="Link_1"/>
        <child link="camera_link_base_right"/>
    </joint>

    <link name="camera_link_base_right">
        <inertial>
            <mass value="1e-5"/>
            <origin rpy="0 0 0" xyz="0 -0.02 0.02"/>
            <inertia ixx="1e-6" ixy="0" ixz="0" iyy="1e-6" iyz="0" izz="1e-6"/>
        </inertial>
        <collision>
            <origin rpy="0 0 -1.570" xyz="0 0 0"/> 
            <geometry>
                <box size="0.025 0.025 0.025"/>
            </geometry>
        </collision>
        <visual>
            <origin rpy="0 0 -1.570" xyz="0 0 0"/>
            <geometry>
                <box size="0.025 0.025 0.025"/>
            </geometry>
            <material name="black">
                <color rgba="1 1 1 1"/>
            </material>
        </visual>
    </link> -->

    <!-- <joint name="camera_optical_joint_base_right" type="fixed">
        <parent link="camera_link_base_right"/>
        <child link="camera_link_optical_base_right"/>
        <origin xyz="0 0 0" rpy="-1.5707963 0 -1.5707963"/>
    </joint>

    <link name="camera_link_optical_base_right"></link> -->


    <!-- Add camera optical frame with standard computer vision orientation -->
    <!-- <joint name="camera_optical_joint" type="fixed">
    <origin rpy="-1.5707963 0 -1.5707963" xyz="0.0 0.0 0.0"/>
    <parent link="camera_link"/>
    <child link="camera_optical_link"/>
  </joint>
  <link name="camera_optical_link">
  </link> -->

    <link name="Link_2">
        <inertial>
            <origin xyz="0.0039680486454326 0.217484463914853 0.0226240865668204" rpy="0 0 0" />
            <mass value="0.696616277349387" />
            <inertia ixx="0.00867448656134372" ixy="-1.47433814721146E-05" ixz="-2.58004719868412E-09" iyy="0.000403909974825226" iyz="4.87457749930191E-10" izz="0.00872331013959563" />
        </inertial>
        <visual>
            <origin xyz="0 0 0" rpy="0 0 0" />
            <geometry>
                <mesh filename="package://Arm_Urdf/meshes/Link_2.STL" />
            </geometry>
            <material name="">
                <color rgba="0.898039215686275 0.917647058823529 0.929411764705882 1" />
            </material>
        </visual>

        <collision>
            <origin xyz="0 0 0" rpy="0 0 0" />
            <geometry>
                <mesh filename="package://Arm_Urdf/meshes/Link_2.STL" />
            </geometry>
        </collision>
    </link>

    <joint name="Joint_2" type="revolute">
        <origin xyz="-0.0299999999999999 0.114999999999999 0.0450000000000004" rpy="1.5707963267949 0 1.5707963267949" />
        <parent link="Link_1" />
        <child link="Link_2" />
        <axis xyz="0 0 -1" />
        <limit lower="-1.570" upper="1.57" effort="50" velocity="5" />
        <dynamics damping="0.5" friction="0.1"/>
    </joint>
    <link name="Link_3">
        <inertial>
            <origin xyz="0.0159437073865258 0.0399778044221051 0.000165930315254696" rpy="0 0 0" />
            <mass value="0.504770554183686" />
            <inertia ixx="0.000217244480297463" ixy="-3.21837262846156E-05" ixz="1.05210946108765E-18" iyy="0.00156585111098331" iyz="-2.22489334296148E-07" izz="0.00165843696768256" />
        </inertial>
        <visual>
            <origin xyz="0 0 0" rpy="0 0 0" />
            <geometry>
                <mesh filename="package://Arm_Urdf/meshes/Link_3.STL" />
            </geometry>
            <material name="">
                <color rgba="0.898039215686275 0.917647058823529 0.929411764705882 1" />
            </material>
        </visual>
        <collision>
            <origin xyz="0 0 0" rpy="0 0 0" />
            <geometry>
                <mesh filename="package://Arm_Urdf/meshes/Link_3.STL" />
            </geometry>
        </collision>
    </link>
    <joint name="Joint_3" type="revolute">
        <origin xyz="0 0.45 0.0301042347134805" rpy="0 0 0" />
        <parent link="Link_2" />
        <child link="Link_3" />
        <axis xyz="0 0 -1" />
        <limit lower="-1.00" upper="1.00" effort="50" velocity="5" />
        <dynamics damping="0.5" friction="0.10"/>
    </joint>
    <link name="Link_4">
        <inertial>
            <origin xyz="0.00264954149583032 0.00172553243144247 -0.256832165659789" rpy="0 0 0" />
            <mass value="0.739327182415655" />
            <inertia ixx="0.00148243608862922" ixy="-2.48364112345643E-07" ixz="-1.03283742873345E-07" iyy="0.00143272022403598" iyz="5.43371774425014E-06" izz="0.000309171282333902" />
        </inertial>
        <visual>
            <origin xyz="0 0 0" rpy="0 0 0" />
            <geometry>
                <mesh filename="package://Arm_Urdf/meshes/Link_4.STL" />
            </geometry>
            <material name="">
                <color rgba="0.898039215686275 0.917647058823529 0.929411764705882 1" />
            </material>
        </visual>
        <collision>
            <origin xyz="0 0 0" rpy="0 0 0" />
            <geometry>
                <mesh filename="package://Arm_Urdf/meshes/Link_4.STL" />
            </geometry>
        </collision>
    </link>
    <joint name="Joint_4" type="revolute">
        <origin xyz="0.0424999999995649 0.0399572355492206 0.000401106428108807" rpy="0 1.5707963267949 0" />
        <parent link="Link_3" />
        <child link="Link_4" />
        <axis xyz="0 0 1" />
        <limit lower="-3.17" upper="3.17" effort="50" velocity="5" />
        <dynamics damping="0.5" friction="0.1"/>
    </joint>
    <link name="Link_5">
        <inertial>
            <origin xyz="-3.16885406803635E-13 -0.00981221204232474 0.00224881068265584" rpy="0 0 0" />
            <mass value="0.0763957117756357" />
            <inertia ixx="3.83083473961502E-05" ixy="3.17560849313193E-16" ixz="2.01751281333396E-17" iyy="4.00747073118559E-05" iyz="-4.08795655667539E-17" izz="2.28558461666204E-05" />
        </inertial>
        <visual>
            <origin xyz="0 0 0" rpy="0 0 0" />
            <geometry>
                <mesh filename="package://Arm_Urdf/meshes/Link_5.STL" />
            </geometry>
            <material name="">
                <color rgba="0.898039215686275 0.917647058823529 0.929411764705882 1" />
            </material>
        </visual>
        <collision>
            <origin xyz="0 0 0" rpy="0 0 0" />
            <geometry>
                <mesh filename="package://Arm_Urdf/meshes/Link_5.STL" />
            </geometry>
        </collision>
    </link>
    <joint name="Joint_5" type="revolute">
        <origin xyz="-0.00649999999995384 0 -0.383499999999573" rpy="0 1.5707963267949 0" />
        <parent link="Link_4" />
        <child link="Link_5" />
        <axis xyz="0 0 1" />
        <limit lower="-0.065" upper="3.17" effort="50" velocity="5" />
        <dynamics damping="0.5" friction="0.10"/>
    </joint>
    <link name="Link_6">
        <inertial>
            <origin xyz="0.0135613456334422 -0.0265420912928073 0.0301408683365089" rpy="0 0 0" />
            <mass value="0.252745339929401" />
            <inertia ixx="0.000195067050160143" ixy="2.5230798194637E-05" ixz="6.2223463246536E-05" iyy="0.000903429764130728" iyz="-1.48620728173967E-05" izz="0.000993241239460536" />
        </inertial>
        <visual>
            <origin xyz="0 0 0" rpy="0 0 0" />
            <geometry>
                <mesh filename="package://Arm_Urdf/meshes/Link_6.STL" />
            </geometry>
            <material name="">
                <color rgba="0.898039215686275 0.917647058823529 0.929411764705882 1" />
            </material>
        </visual>
        <collision>
            <origin xyz="0 0 0" rpy="0 0 0" />
            <geometry>
                <mesh filename="package://Arm_Urdf/meshes/Link_6.STL" />
            </geometry>
        </collision>
    </link>
    <joint name="Joint_6" type="revolute">
        <origin xyz="0 -0.0199999999999119 0.00650441096988074" rpy="1.5707963267949 -1.5707963267949 0" />
        <parent link="Link_5" />
        <child link="Link_6" />
        <axis xyz="0 0 1" />
        <limit lower="-3.17" upper="3.17" effort="50" velocity="5" />
        <dynamics damping="0.5" friction="0.10"/>
    </joint>
    <joint name="camera_joint2" type="fixed">
        <axis xyz="0 1 0"/>
        <origin rpy="0 -1.57 1.57" xyz="0.0 -0.082 0.012"/>
        <parent link="Link_6"/>
        <child link="camera_link2"/>
    </joint>
    <link name="camera_link2">
        <inertial>
            <mass value="1e-5"/>
            <origin rpy="0 0 0" xyz="0.0 -0.082 0.012"/>
            <inertia ixx="1e-6" ixy="0" ixz="0" iyy="1e-6" iyz="0" izz="1e-6"/>
        </inertial>
        <collision>
            <origin rpy="0 0 0" xyz="0 0 0"/>
            <geometry>
                <box size="0.025 0.025 0.025"/>
            </geometry>
        </collision>
        <visual>
            <origin rpy="0 0 0" xyz="0 0 0"/>
            <geometry>
                <box size="0.025 0.025 0.025"/>
            </geometry>
            <material name="black"/>
        </visual>
    </link>
    <link name="Gripper_1">
        <inertial>
            <origin xyz="0.0204003994868009 0.0334764538131697 0.063230838084357" rpy="0 0 0" />
            <mass value="0.137742996071251" />
            <inertia ixx="5.29911205654385E-05" ixy="-1.12421617290008E-06" ixz="-2.49192530851908E-06" iyy="7.19109477567595E-05" iyz="-1.11448830844399E-05" izz="6.43837075224686E-05" />
        </inertial>
        <visual>
            <origin xyz="0 0 0" rpy="0 0 0" />
            <geometry>
                <mesh filename="package://Arm_Urdf/meshes/Gripper_1.STL" />
            </geometry>
            <material name="">
                <color rgba="0.776470588235294 0.756862745098039 0.737254901960784 1" />
            </material>
        </visual>
        <collision>
            <origin xyz="0 0 0" rpy="0 0 0" />
            <geometry>
                <mesh filename="package://Arm_Urdf/meshes/Gripper_1.STL" />
            </geometry>
        </collision>
    </link>
    <joint name="Finger_1" type="prismatic">
        <origin xyz="0.00534999999999963 -0.0390954035350414 0.0487946645392847" rpy="1.5707963267949 0 1.5707963267949" />
        <parent link="Link_6" />
        <child link="Gripper_1" />
        <axis xyz="0 0 -1" />
        <limit lower="0" upper="0.0355" effort="100" velocity="10" />
        <dynamics damping="0.5" friction="0.1" />
    </joint>
    <link name="Gripper_2">
        <inertial>
            <origin xyz="-0.0145607269510522 0.0386349312979239 -0.0735904539348097" rpy="0 0 0" />
            <mass value="0.153524806277575" />
            <inertia ixx="5.29298080219616E-05" ixy="1.90492712258527E-06" ixz="-3.08938743832324E-06" iyy="7.11796554106749E-05" iyz="1.10602892279447E-05" izz="6.37754371292033E-05" />
        </inertial>
        <visual>
            <origin xyz="0 0 0" rpy="0 0 0" />
            <geometry>
                <mesh filename="package://Arm_Urdf/meshes/Gripper_2.STL" />
            </geometry>
            <material name="">
                <color rgba="0.776470588235294 0.756862745098039 0.737254901960784 1" />
            </material>
        </visual>
        <collision>
            <origin xyz="0 0 0" rpy="0 0 0" />
            <geometry>
                <mesh filename="package://Arm_Urdf/meshes/Gripper_2.STL" />
            </geometry>
        </collision>
    </link>
    <joint name="Finger_2" type="prismatic">
        <dynamics damping="0.5" friction="0.1" />
        <origin xyz="0.00534999999999963 -0.0390954035350414 0.0487946645392847" rpy="1.5707963267949 0 1.5707963267949" />
        <parent link="Link_6" />
        <child link="Gripper_2" />
        <axis xyz="0 0 -1" />
        <limit lower="-0.0355" upper="0" effort="100" velocity="10" />
    </joint>
    <transmission name="trans_Joint_1">
        <type>transmission_interface/SimpleTransmission</type>
        <joint name="Joint_1">
            <hardwareInterface>hardware_interface/PositionJointInterface</hardwareInterface>
        </joint>
        <actuator name="Link_1_motor">
            <hardwareInterface>hardware_interface/PositionJointInterface</hardwareInterface>
            <mechanicalReduction>1</mechanicalReduction>
        </actuator>
    </transmission>
    <transmission name="trans_Joint_2">
        <type>transmission_interface/SimpleTransmission</type>
        <joint name="Joint_2">
            <hardwareInterface>hardware_interface/PositionJointInterface</hardwareInterface>
        </joint>
        <actuator name="Link_2_motor">
            <hardwareInterface>hardware_interface/PositionJointInterface</hardwareInterface>
            <mechanicalReduction>1</mechanicalReduction>
        </actuator>
    </transmission>
    <transmission name="trans_Joint_3">
        <type>transmission_interface/SimpleTransmission</type>
        <joint name="Joint_3">
            <hardwareInterface>hardware_interface/PositionJointInterface</hardwareInterface>
        </joint>
        <actuator name="Link_3_motor">
            <hardwareInterface>hardware_interface/PositionJointInterface</hardwareInterface>
            <mechanicalReduction>1</mechanicalReduction>
        </actuator>
    </transmission>
    <transmission name="trans_Joint_4">
        <type>transmission_interface/SimpleTransmission</type>
        <joint name="Joint_4">
            <hardwareInterface>hardware_interface/PositionJointInterface</hardwareInterface>
        </joint>
        <actuator name="Link_4_motor">
            <hardwareInterface>hardware_interface/PositionJointInterface</hardwareInterface>
            <mechanicalReduction>1</mechanicalReduction>
        </actuator>
    </transmission>
    <transmission name="trans_Joint_5">
        <type>transmission_interface/SimpleTransmission</type>
        <joint name="Joint_5">
            <hardwareInterface>hardware_interface/PositionJointInterface</hardwareInterface>
        </joint>
        <actuator name="Link_5_motor">
            <hardwareInterface>hardware_interface/PositionJointInterface</hardwareInterface>
            <mechanicalReduction>1</mechanicalReduction>
        </actuator>
    </transmission>
    <transmission name="trans_Joint_6">
        <type>transmission_interface/SimpleTransmission</type>
        <joint name="Joint_6">
            <hardwareInterface>hardware_interface/PositionJointInterface</hardwareInterface>
        </joint>
        <actuator name="Link_6_motor">
            <hardwareInterface>hardware_interface/PositionJointInterface</hardwareInterface>
            <mechanicalReduction>1</mechanicalReduction>
        </actuator>
    </transmission>
    <transmission name="trans_Finger_1">
        <type>transmission_interface/SimpleTransmission</type>
        <joint name="Finger_1">
            <hardwareInterface>hardware_interface/PositionJointInterface</hardwareInterface>
        </joint>
        <actuator name="Gripper_1_motor">
            <hardwareInterface>hardware_interface/PositionJointInterface</hardwareInterface>
            <mechanicalReduction>1</mechanicalReduction>
        </actuator>
    </transmission>
    <transmission name="trans_Finger_2">
        <type>transmission_interface/SimpleTransmission</type>
        <joint name="Finger_2">
            <hardwareInterface>hardware_interface/PositionJointInterface</hardwareInterface>
        </joint>
        <actuator name="Gripper_2_motor">
            <hardwareInterface>hardware_interface/PositionJointInterface</hardwareInterface>
            <mechanicalReduction>1</mechanicalReduction>
        </actuator>
    </transmission>
    <gazebo>
        <plugin name="gazebo_ros_control" filename="libgazebo_ros_control.so">
            <robotNamespace>/</robotNamespace>
        </plugin>
    </gazebo>


    <gazebo reference="camera_link_optical_base_left">
        <sensor type="camera" name="camera_base">
            <visualize>true</visualize>
            <update_rate>30.0</update_rate>
            <camera name="head">
                <horizontal_fov>1.3962634</horizontal_fov>
                <image>
                    <width>800</width>
                    <height>800</height>
                    <format>R8G8B8</format>
                </image>
                <clip>
                    <near>0.02</near>
                    <far>300</far>
                </clip>
                <noise>
                    <type>gaussian</type>
                    <!-- Noise is sampled independently per pixel on each frame.
                That pixel's noise value is added to each of its color
                channels, which at that point lie in the range [0,1]. -->
                    <mean>0.0</mean>
                    <stddev>0.007</stddev>
                </noise>
            </camera>
            <plugin name="camera_controller" filename="libgazebo_ros_camera.so">
                <alwaysOn>true</alwaysOn>
                <updateRate>0.0</updateRate>
                <cameraName>camera_base_left</cameraName>
                <imageTopicName>image_raw</imageTopicName>
                <cameraInfoTopicName>camera_info</cameraInfoTopicName>
                <frameName>camera_link_base_left</frameName>
                <hackBaseline>0.07</hackBaseline>
                <distortionK1>0.0</distortionK1>
                <distortionK2>0.0</distortionK2>
                <distortionK3>0.0</distortionK3>
                <distortionT1>0.0</distortionT1>
                <distortionT2>0.0</distortionT2>
            </plugin>
        </sensor>
    </gazebo>


    <gazebo reference="camera_link2">
        <sensor type="camera" name="camera_gripper">
            <visualize>true</visualize>
            <update_rate>30.0</update_rate>
            <camera name="head">
                <horizontal_fov>1.3962634</horizontal_fov>
                <image>
                    <width>800</width>
                    <height>800</height>
                    <format>R8G8B8</format>
                </image>
                <clip>
                    <near>0.02</near>
                    <far>300</far>
                </clip>
                <noise>
                    <type>gaussian</type>
                    <!-- Noise is sampled independently per pixel on each frame.
                That pixel's noise value is added to each of its color
                channels, which at that point lie in the range [0,1]. -->
                    <mean>0.0</mean>
                    <stddev>0.007</stddev>
                </noise>
            </camera>
            <plugin name="camera_controller" filename="libgazebo_ros_camera.so">
                <alwaysOn>true</alwaysOn>
                <updateRate>0.0</updateRate>
                <cameraName>camera_gripper</cameraName>
                <imageTopicName>image_raw</imageTopicName>
                <cameraInfoTopicName>camera_info</cameraInfoTopicName>
                <frameName>camera_link2</frameName>
                <hackBaseline>0.07</hackBaseline>
                <distortionK1>0.0</distortionK1>
                <distortionK2>0.0</distortionK2>
                <distortionK3>0.0</distortionK3>
                <distortionT1>0.0</distortionT1>
                <distortionT2>0.0</distortionT2>
            </plugin>
        </sensor>
    </gazebo>

    <gazebo reference="camera_link_optical_base_right">
        <sensor type="camera" name="camera_base_2">
            <visualize>true</visualize>
            <update_rate>30.0</update_rate>
            <camera name="head">
                <horizontal_fov>1.3962634</horizontal_fov>
                <image>
                    <width>800</width>
                    <height>800</height>
                    <format>R8G8B8</format>
                </image>
                <clip>
                    <near>0.02</near>
                    <far>300</far>
                </clip>
                <noise>
                    <type>gaussian</type>
                    <!-- Noise is sampled independently per pixel on each frame.
                That pixel's noise value is added to each of its color
                channels, which at that point lie in the range [0,1]. -->
                    <mean>0.0</mean>
                    <stddev>0.007</stddev>
                </noise>
            </camera>
            <plugin name="camera_controller" filename="libgazebo_ros_camera.so">
                <alwaysOn>true</alwaysOn>
                <updateRate>0.0</updateRate>
                <cameraName>camera_base_2</cameraName>
                <imageTopicName>image_raw</imageTopicName>
                <cameraInfoTopicName>camera_info</cameraInfoTopicName>
                <frameName>camera_base_right</frameName>
                <hackBaseline>0.07</hackBaseline>
                <distortionK1>0.0</distortionK1>
                <distortionK2>0.0</distortionK2>
                <distortionK3>0.0</distortionK3>
                <distortionT1>0.0</distortionT1>
                <distortionT2>0.0</distortionT2>
            </plugin>
        </sensor>
    </gazebo>
</robot>
